### PR TITLE
Introducing JSON log

### LINKED ISSUE #149: Log results in JSON format
Sorry if this feature already exists, but I've looked and so far found nothing. So, my question is, is it possible to get the results from phploc in JSON format?

### PR DESCRIPTION
**Pull request by kubawerlos**
Resolves #149

### CODE REVIEW

**Comment by kubawerlos** [comment]
@sebastianbergmann any chance for merge?

File: src/CLI/Command.php [hunk 1]
```diff
@@ -97,10 +104,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         if (!$count) {
             $output->writeln('No files found to scan');
-            exit(0);
+            return 0;
         }
 
-        $printer = new Text;
+        $printer = new Text();
```
Comment (sebastianbergmann) [inline src/CLI/Command.php]: Please do not make changes that are not relevant to the purpose of the pull request.

Comment (kubawerlos) [reply src/CLI/Command.php]: Ok, I have extracted the changes to: https://github.com/sebastianbergmann/phploc/pull/176 so if you merge it first it won't show here.

Comment (sebastianbergmann) [reply src/CLI/Command.php]: The point I wanted to make (and apparently failed to make) is that I do not want those changes.

Comment (kubawerlos) [reply src/CLI/Command.php]: Alright, not sure why not to have brackets, but done.

Comment (sebastianbergmann) [reply src/CLI/Command.php]: That should make us even, as I do not understand why people want JSON.

**Pull request merged**